homeassistant/components/generic/config_flow.py

```diff
@@ -412,15 +412,13 @@ async def async_step_user_confirm(
                 title=self.title, data={}, options=self.user_input
             )
         register_preview(self.hass)
-        preview_url = f"/api/generic/preview_flow_image/{self.flow_id}?t={datetime.now().isoformat()}"
```

Comment:
Can you explain with a TL;DR (or in the PR description) of what this part of the chagne is about?

Reply:
This preview URL is the 'old' way of doing a preview image. The Image URL was passed as a description placeholder into the next step of the config flow.
In the 'new' way, both the still and stream preview are part of the javascript generated section in the lower part of the config flow pane. In that case the URL is communicated via websocket, so this is no longer needed.